pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -270,6 +270,11 @@ func deploymentOnNode(c *Cluster, osd OSDInfo, nodeName string, config *provisio
 		return nil, errors.Wrapf(err, "failed to generate deployment for %s", osdLongName)
 	}
 
+	err = prepareForDeployment(c, osdProps, osd, osdLongName)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to apply deployment fixup for %s", osdLongName)
```

Comment:
fixup? Also please use  `%q` instead of `%s`

Reply:
@leseb used "%s" to follow same pattern as rest of this error-formatting in this function. Note that "%q" is already used to format 'osdLongName'